lib/pharos/cluster_manager.rb

```diff
@@ -101,6 +101,7 @@ def apply_phases
 
       # master is now configured and can be used
       # configure essential services
+      apply_phase(Phases::ConfigureLicenseEnforcer, master_only)
```

Comment:
Could phases be injected from pro/ee versions? /cc @kke

Reply:
Well, we could trick apply_phase:

```ruby
class ClusterManager
  def apply_phase(klass, hosts, **opts)
    super
    if klass == Phases::FooFoo
      super(Phases::ConfigureLicenseEnforcer, [config.master_host])
    end
  end
end
```

Reply:
we decided to go with `phase.enabled?` way...